status.xrap1dx.json

```
@@ -0,0 +1,13 @@
+{
+  "description": "making my status page!!!",
+  "repo": "https://github.com/xrap1dx?tab=repositories",
+  "owner": {
+    "username": "xrap1dx",
+    "email": "sirrapid155@gmail.com"
+  },
+  "record": {
+    "CNAME": {
+      "status": "statuspage.betteruptime.com"
+    }
```

CNAME should be a string
```suggestion
    "CNAME": "statuspage.betteruptime.com"
```